src/organisms/Trips.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect } from 'react';
+import Grid from '@material-ui/core/Grid';
+import BusCard from '../Molecules/BusCard.js';
+import InputFeild from '../Atoms/InputFeild.js';
+import RefreshIcon from '@material-ui/icons/Refresh';
+import Custombutton from '../Atoms/Custombutton.js';
+
+const Trips = (props) => {
+    const trips = props.tripdetails
+    const tempTrips = [...trips]
+    const [tripData, setTripData] = React.useState([]);
+    const [flag, setFlag] = React.useState('')
+    const [company, setCompany] = React.useState('')
+    const [temp] = React.useState(tempTrips);
+
+    useEffect(() => {
+        if (company) {
```

Comment:
i see default value of company is '' which will never be false, what is the purpose of this statement